modules/azure-oracle-adbs/README.md

```diff
@@ -0,0 +1,57 @@
+| `nw_delegated_subnet_name`      | Oracle delegate subnet name                                                                                                                                                                                                   | string | n/a               |   yes    |
+| `db_resource_group`             | ADBS resource group name                                                                                                                                                                                                      | string | n/a               |   yes    |
+| `db_name`                       | Database name. The name must begin with an alphabetic character and can contain a maximum of 14 alphanumeric characters. Special characters are not permitted                                                                 | string | n/a               |   yes    |
+| `db_admin_password`             | Password must be between 12 and 30 characters long, and must contain at least 1 uppercase, 1 lowercase, and 1 numeric character. It cannot contain the double quote symbol (“) or the username “admin”, regardless of casing. | string | n/a               |   yes    |
+| `db_ecpu_count`                 | Number of CPU cores to be made available to the database                                                                                                                                                                      | number | n/a               |   yes    |
+| `db_storage_in_gb`              | Size, in gigabytes, of the data volume that will be created and attached to the database                                                                                                                                      | number | n/a               |   yes    |
+| `db_version`                    | Oracle Database version for Autonomous Database                                                                                                                                                                               | string | "19c"             |          |
+| `db_license_model`              | License model either LicenseIncluded or BringYourOwnLicense "                                                                                                                                                                 | string | "LicenseIncluded" |          |
+| `db_workload`                   | Autonomous Database workload type                                                                                                                                                                                             | string | "DW"              |          |
```

Comment:
Shall we also provide the enum value for ATP?

Reply:
We are already providing link to database docs. 
Terraform template and ADBS vars are detached, if we create enum we will have to keep enum up to date.